app/lib/dags/experimental.json.erb

```diff
@@ -38,7 +38,11 @@
       "taxid_locations_combined.json"
     ],
     "alignment_viz_out": ["align_viz.summary"],
+    <% if attribute_dict[:pipeline_version].to_f >= 3.8 %>
```

Comment:
I think if we start adding pipeline version conditions to this file, it will quickly become unmanageable. In the past, we've usually just made a clean break and deployed the new idseq-dag version at the same time as the new idseq-web version to keep the two in sync. There is currently no expectation that the web will be backwards-compatible with older idseq-dag versions, so you don't need to allow for it here.

In other words: please get rid of the if-else here and just make sure to coordinate the idseq-dag and idseq-web deployments.

Ignore me if you're just writing this condition for some testing in staging. My comment applies to the final code that goes into master after you have experimented and tested it.

Reply:
@cdebourcy I had advised her to do this because it seemed like a safer way to deploy. If you have this if-else wrapper, then the dag will be valid both before and after the idseq_dag feature branch is merged into master. The plan is to then remove the if-else wrapper once the dag change is deployed and stable, so it will not be there long-term.

The reason it feels safer is that you don't have to coordinate the deploy of the two services, with a risk that if a user submits a sample at a bad time, it might fail.

My approach does involve deploying the if-else to production and then removing it later.

Let me know if you prefer your way, and I can change the idseq dag quick start docs.